src/lib/plugin.js

```diff
@@ -130,7 +131,7 @@ class FiveBellsLedger extends EventEmitter2 {
 
     // Resolve ledger metadata
     const ledgerMetadata = yield this._fetchLedgerMetadata()
-    this.accountUriTemplate = ledgerMetadata.urls.account
+    this.urls = ledgerMetadata.urls
```

Comment:
Addressed in https://github.com/interledger/js-ilp-plugin-bells/pull/52/commits/71ee1b7e82db7817f3be32da6345943605cb7de8